MarvelApiClient/src/main/java/com/karumi/MarvelApiClient/model/ComicResource.java

```diff
@@ -0,0 +1,19 @@
+/*
+ *   Copyright (C) 2015 Karumi.
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package com.karumi.marvelapiclient.model;
+
+public class ComicResource extends MarvelResource {
+}
```

Comment:
IMHO the only thin you are going to generate in a developer mind is a brain fuck xD. Using this code you have a getter for this class but once you get the instance you can't use it :( Definitively we should remove or finish these classes.